beamer/tests/util.py

```diff
@@ -28,6 +29,14 @@ def alloc_accounts(n):
     return tuple(_alloc_account() for _ in range(n))
 
 
+def alloc_whitelisted_accounts(n, contracts):
```

Comment:
This assumes each contracts is owned by the default address. Maybe that's an assumption you have everywhere in the tests, in which case it is fine.